product_docs/docs/efm/4/04_configuring_efm/05_using_vip_addresses.mdx

```diff
@@ -12,9 +12,12 @@ legacyRedirectsGenerated:
 
 Failover Manager uses the `efm_address` script to assign or release a virtual IP address.
 
-!!! Note
+!!! Note "Cloud provider support and alternatives"
     Virtual IP addresses aren't supported by many cloud providers. In those environments, use another mechanism, such as an elastic IP address on AWS, that can be changed when needed by a fencing or post-promotion script.
 
+!!! Note "Behavior when the agent is stopped"
+    Failover Manager will not drop the Virtual IP address (if used) from the primary node when the agent is stopped. As a convenience for testing, the primary agent will acquire the VIP at startup if the node does not already have it, but otherwise starting and stopping Failover Manager has no effect on the node having the virtual IP address.
```

Comment:
I suspect some of the confusion here ties into the final note in this file (line 156):

> The primary agent drops the VIP as needed during a failover, and Failover Manager verifies that the VIP is no longer available before promoting a standby. A failure of the bind address network leads to primary isolation and failover.

We should probably try to clarify that. Perhaps the two notes could be combined into a couple of paragraphs that lay out more clearly the scenarios in which EFM will drop / acquire a VIP?

Reply:
I'd rather make the smallest number of changes possible. Stopping efm != a failure -- I just want to let people know stopping efm doesn't break the cluster, and that's all separate from efm behavior while running.